modules/serverless-installing-cli-linux-ibm-power-rpm.adoc

```diff
@@ -0,0 +1,43 @@
+[id="installing-cli-linux-ibm-power-rpm_{context}"]
+= Installing the Knative CLI for Linux on IBM Power Systems using an RPM
+
+For {op-system-base-full}, you can install the `kn` CLI as an RPM if you have an active {product-title} subscription on your Red Hat account.
+
+.Procedure
+
+. Register with a subscription service during the firstboot process:
++
+[source,terminal]
+----
+# subscription-manager register
+----
+
+. Refresh the subscription-manager:
```

Comment:
I'd either use `subscription-manager` or Subscription Manager maybe if this is a "product"? Not sure which it should be, might be worth checking with someone on the OCP team :)